src/azul/service/manifest_service.py

```diff
@@ -113,6 +113,15 @@ class ManifestFormat(Enum):
 
 @attr.s(auto_attribs=True, kw_only=True, frozen=True)
 class Manifest:
+    """
+    A collection of details for a prepared manifest.
+
+    Attributes:
+        location    A URL to the manifest file.
```

Comment:
There must be a better way to document attributes than to brittly duplicate their names.

Reply:
With no other class attributes documented in the Azul codebase to use as reference, I'm not sure what syntax would be preferable here. [PEP-244](https://www.python.org/dev/peps/pep-0224/) sounds like an ideal solution however this is a rejected PEP.

Reply:
> With no other class attributes documented in the Azul codebase to use as reference

Not true. Look for `IndexName`

https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute